supplements/eberron-rising-from-the-last-war/class-artificer.xml

```diff
@@ -1644,4 +1650,111 @@
+		</setters> 
+		<rules>
+			<stat name="companion:ac" value="13" bonus="base" />
+			<stat name="companion:hp:max" value="1" />
+			<stat name="companion:hp:max" value="intelligence:modifier" />
+			<stat name="companion:hp:max" value="level:artificer" />
+			<stat name="companion:speed" value="20" bonus="base" />
+			<stat name="companion:speed:fly" value="40" bonus="base" />
+			<stat name="companion:proficiency" value="proficiency" bonus="base" />
+			<stat name="companion:proficiency" value="-2" bonus="base companion PB removal"/>
+			<stat name="companion:stealths:proficiency" value="companion:proficiency" bonus="proficiency" />
+			<stat name="companion:perception:proficiency" value="companion:proficiency" bonus="proficiency" />
+			<stat name="companion:perception:proficiency" value="companion:proficiency" bonus="double" />
+			<stat name="companion:dexterity:save:proficiency" value="companion:proficiency" bonus="proficiency" />
+		</rules> 
+	</element> 
+	<element name="Evasion" type="Companion Trait" source="Eberron: Rising from the Last War" id="ID_WOTC_ERLW_COMPANION_TRAIT_HOMUNCULUS_SERVANT_EVASION">
+		<description>
+			<p>If the homunculus is subjected to an effect that allows it to make a Dexterity saving throw to take only half damage, it instead takes no damage if it succeeds on the saving throw, and only half damage if it fails. It can't use this trait if it's incapacitated. </p>
+		</description>
+		<sheet>
+			<description>If the homunculus is subjected to an effect that allows it to make a Dexterity saving throw to take only half damage, it instead takes no damage if it succeeds on the saving throw, and only half damage if it fails. It can't use this trait if it's incapacitated. </description> 
+		</sheet> 
+	</element>
+	<element name="Force Strike" type="Companion Action" source="Eberron: Rising from the Last War" id="ID_WOTC_ERLW_COMPANION_ACTION_HOMUNCULUS_SERVANT_FORCE_STRIKE">
+		<description>
+			<p>Ranged Weapon Attack: your spell attack modifier to hit, range 30 ft., one target you can see. Hit: 1d4 + PB force damage. </p>
+		</description>
+		<sheet>
+			<description>Ranged Weapon Attack: your spell attack modifier to hit, range 30 ft., one target you can see. Hit: 1d4 + PB force damage. </description>
+		</sheet> 
+	</element>
+	<element name="Channel Magic" type="Companion Reaction" source="Eberron: Rising from the Last War" id="ID_WOTC_ERLW_COMPANION_REACTION_HOMUNCULUS_SERVANT_CHANNEL_MAGIC">
+		<description>
+			<p> The homunculus delivers a spell you cast that has a range of touch. The homunculus must be within 120 feet of you. </p>
+		</description>
+		<sheet>
+			<description> The homunculus delivers a spell you cast that has a range of touch. The homunculus must be within 120 feet of you. </description>
+		</sheet> 
+	</element>
+
+	<element name="Homunculus Servant" type="Information" source="Eberron: Rising from the Last War" id="ID_WOTC_ERLW_INFORMATION_HOMUNCULUS_SERVANT_STATBLOCK">
```

Comment:
why do we need this? Why can't you just use Companion element for a reference?

Reply:
![image](https://user-images.githubusercontent.com/15676362/123126920-6b04b280-d40f-11eb-9a73-53263312d731.png)
Using Just the companion for the div gives a very different result